{"uid":"cc08db5d11eec6b7","name":"Check App build and running","fullName":"Check App build and running","historyId":"bba9896da91f8193b73d61f40f2c0f74","time":{"start":1698655398918,"stop":1698655399534,"duration":616},"status":"broken","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ScriptExternalLoadError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at webpack/container/reference/component-app (http://localhost:3002/main.js:11:25)\n    at __webpack_require__ (http://localhost:3002/main.js:72:41)\n    at handleFunction (http://localhost:3002/main.js:247:31)\n    at <unknown> (http://localhost:3002/main.js:266:15)\n    at Array.forEach (<anonymous>)\n    at __webpack_require__.f.remotes (http://localhost:3002/main.js:229:36)\n    at <unknown> (http://localhost:3002/main.js:113:40)\n    at Array.reduce (<anonymous>)\n    at __webpack_require__.e (http://localhost:3002/main.js:112:67)\n    at <unknown> (http://localhost:3002/main.js:423:21)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ScriptExternalLoadError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at webpack/container/reference/component-app (http://localhost:3002/main.js:11:25)\n    at __webpack_require__ (http://localhost:3002/main.js:72:41)\n    at handleFunction (http://localhost:3002/main.js:247:31)\n    at <unknown> (http://localhost:3002/main.js:266:15)\n    at Array.forEach (<anonymous>)\n    at __webpack_require__.f.remotes (http://localhost:3002/main.js:229:36)\n    at <unknown> (http://localhost:3002/main.js:113:40)\n    at Array.reduce (<anonymous>)\n    at __webpack_require__.e (http://localhost:3002/main.js:112:67)\n    at <unknown> (http://localhost:3002/main.js:423:21)","steps":[{"name":"\"before each\" hook","time":{"start":1698655398919,"stop":1698655399534,"duration":615},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ScriptExternalLoadError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at webpack/container/reference/component-app (http://localhost:3002/main.js:11:25)\n    at __webpack_require__ (http://localhost:3002/main.js:72:41)\n    at handleFunction (http://localhost:3002/main.js:247:31)\n    at <unknown> (http://localhost:3002/main.js:266:15)\n    at Array.forEach (<anonymous>)\n    at __webpack_require__.f.remotes (http://localhost:3002/main.js:229:36)\n    at <unknown> (http://localhost:3002/main.js:113:40)\n    at Array.reduce (<anonymous>)\n    at __webpack_require__.e (http://localhost:3002/main.js:112:67)\n    at <unknown> (http://localhost:3002/main.js:423:21)","steps":[{"name":"visit http://localhost:3002","time":{"start":1698655399041,"stop":1698655399534,"duration":493},"status":"failed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"uncaught exception ScriptExternalLoadError: Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app","time":{"start":1698655399532,"stop":1698655399532,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":0,"hasContent":true}],"attachments":[{"uid":"907f40713bb6e591","name":"e2e/checkApplication.cy.ts:2023-10-30T08:43:19.254Z:attempt-1","source":"907f40713bb6e591.png","type":"image/png","size":165896},{"uid":"835d63b1ad9dfffd","name":"video recording","source":"835d63b1ad9dfffd.mp4","type":"video/mp4","size":808693}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":2,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.checkApplication.cy.ts"},{"name":"parentSuite","value":"Complete React case"},{"name":"suite","value":"Check App1"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"cc08db5d11eec6b7.json","parameterValues":[]}